Training_analysis.py

```diff
@@ -108,8 +109,12 @@
 #
 # This notebook has been updated to support the most recent setups. Most of the mentioned projects above are no longer compatible with AWS DeepRacer Console anyway so do consider moving to the ones actively maintained.
 #     
+# **DeepRacer On The Spot (or other S3 log hosting)**
+#
+# Don't use the section directly below, instead use the following section that starts '#login to AWS'
```

Comment:
Don't do separate complete sections; for now I think S3FileHandler can be default.

Reply:
Implemented as requested